modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing_bare_metal/installing-bare-metal-network-customizations.adoc
+
+[id="installation-user-infra-machines-static-network_{context}"]
+= Configuring static networking
+
+If you install {op-system-first} from an ISO image, you can add kernel arguments
+when you boot that image to configure the node's networking.
+The following table describes and illustrates how to use those kernel arguments.
+
+.Configuring static networking
+[source,adoc]
+|===
+|Description |Examples
+
+a|To configure an IP address, either use DHCP (`ip=dhcp`) or set an individual
+static IP address (`ip=<host_ip>`). Then identify the DNS server IP address (`nameserver=<dns_ip>`) on each node. This example sets: +
+
+* The node's IP address to `10.10.10.2` +
+* The gateway address to `10.10.10.254` +
+* The netmask to `255.255.255.0` +
+* The hostname to `core0.example.com` +
+* The DNS server address to `4.4.4.41`
+
+|`ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s0:none nameserver=4.4.4.41` +
+
+|Specify multiple network interfaces by specifying multiple `ip=` entries.
+|`ip=192.168.122.25 ip=192.168.122.26`
+
+|You can combine DHCP
+and static IP configurations on systems with
+multiple network interfaces.
+|`ip=enp1s0:dhcp ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp2s0:none`.
+
+|You can provide multiple DNS servers by adding a `nameserver=` entry for each server
+|`nameserver=1.1.1.1 nameserver=8.8.8.8`.
+
+a|Bonding multiple network interfaces to a single interface is optionally supported
+using the `bond=` option.  In these two examples:
+
+* _name_ is the bonding device name (bond0), _network_interfaces_
+represents a comma-separated list of physical (ethernet) interfaces (`em1,em2`),
+and _options_ is a comma-separated list of bonding options.
+* When you
+create a bonded interface using `bond=`, you must specify how the IP address
+is assigned (either with `ip=` or `dhcp`) and other
```

Comment:
I would drop the `(either with `ip=` or `dhcp`)` portion of this sentence; the examples convey how the IP address is being configured.

Reply:
Okay.

Reply:
Dropped.